package.json

```diff
@@ -38,8 +38,6 @@
     "jest": "^29.7.0",
     "ts-jest": "^29.1.1",
     "typescript": "^5.0.2",
-    "vite": "^4.3.2",
-    "vite-plugin-pwa": "^0.15.1",
```

Comment:
@john-wayne630 Please ensure vite-plugin-pwa is indeed not used in provided files

Reply:
Done!

Reply:
Done!